integrated_channels/blackboard/client.py

```diff
@@ -35,6 +36,11 @@ def __init__(self, enterprise_configuration):
         self.config = apps.get_app_config('blackboard')
         self.session = None
         self.expires_at = None
+        self.gradebook_path = '/learn/api/public/v1/courses/{course_id}/gradebook/columns'
```

Comment:
since these are all constants, maybe make them defined constants up at the top in CAPS? upto you, more of a pref thing. It's possible it will make it easier to also use them in tests when we have to

Reply:
I had been thinking of doing this too

Reply:
At the top or in a separate `integrated channels api paths` file?